### ISSUE TITLE
Automate composite dose processing

### ISSUE THREAD

**Issue by cutright** [cutright/DVH-Analytics-Bokeh#32]
**We intend to add automation of composite DVH calculation based on multiple DICOM RT Dose files**

Because not all treatment planning systems can export a composite plan (e.g., putting all rxs in a single rt plan file), functionality to import all rt plan files for a given study instance UID was added (see 3e3db72a39e92b90780ac2a3a7937e33d12b9b7f).

Currently, DVH Analytics will still only import the latest dose and struct files for a given study instance UID, so the user may need to use other software to generate a single rt dose file of the composite dose.  We plan to do this for you in a future update.

In summary, DVH Analytics assumes:
* All rt plan files in the inbox are pertinent (unless IMPORT_LATEST_PLAN_ONLY is set to True in options.py)
* Only the latest rt plan and rt struct files will be used, and are assumed to reflect a composite
     * All DICOM files for a given study instance UID will still be catalogued, even if their data is not added to the Plans, Beams, Rxs, or DVHs tables
